app/navigation/LeadTrusteeOrganisationNavigator.scala

```diff
@@ -17,46 +17,66 @@
 package navigation
 
 import config.FrontendAppConfig
-import controllers.register.leadtrustee.organisation.routes._
+import controllers.register.leadtrustee.organisation.{routes => rts}
+import controllers.register.leadtrustee.organisation.nonTaxable.{routes => mld5Rts}
 import models.ReadableUserAnswers
 import pages.Page
 import pages.register.leadtrustee.organisation._
+import pages.register.leadtrustee.organisation.nonTaxable.{CountryOfResidenceInTheUkYesNoPage, CountryOfResidencePage}
 import play.api.mvc.Call
 
 class LeadTrusteeOrganisationNavigator extends Navigator {
 
-  override def simpleNavigation(draftId: String): PartialFunction[Page, ReadableUserAnswers => Call] = {
-    case UkRegisteredYesNoPage(index) => _ => NameController.onPageLoad(index, draftId)
-    case NamePage(index) => ua => nameRoute(ua, index, draftId)
-    case UtrPage(index) => _ => AddressUkYesNoController.onPageLoad(index, draftId)
-    case UkAddressPage(index) => _ => EmailAddressYesNoController.onPageLoad(index, draftId)
-    case InternationalAddressPage(index) => _ => EmailAddressYesNoController.onPageLoad(index, draftId)
-    case EmailAddressPage(index) => _ => TelephoneNumberController.onPageLoad(index, draftId)
-    case TelephoneNumberPage(index) => _ => CheckDetailsController.onPageLoad(index, draftId)
+  override def simpleNavigation(draftId: String, fiveMldEnabled: Boolean): PartialFunction[Page, ReadableUserAnswers => Call] = {
```

Comment:
Very picky but can we have `5mld` instead of `fiveMld` to be consistent? Map apply to a few places

Reply:
intellij complains about parameters that start with a number